plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1262,11 +1319,15 @@ public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle
     }
 
     private void appendTableEntries(
+            ColumnMappingMode columnMappingMode,
             long commitVersion,
             TransactionLogWriter transactionLogWriter,
             String tableId,
-            List<DeltaLakeColumnHandle> columns,
+            List<String> columnNames,
+            AtomicInteger newFieldId,
```

Comment:
`newFieldId` and `newColumnName` are effectively empty for all the calls to this method - let's remove them from here.